digital_land/pipeline.py

```diff
@@ -194,13 +203,20 @@ def load_redirect_lookup(self):
             old_entity = row.get("old-entity", "")
             entity = row.get("entity", "")
             status = row.get("status", "")
-            if old_entity and status:
+            if old_entity and entity and status:
                 self.redirect_lookup[old_entity] = {"entity": entity, "status": status}
 
-    def filters(self, resource=""):
-        d = self.filter.get("", {})
+    def filters(self, resource="", endpoints=""):
```

Comment:
the default endpoint here is a string but should it be a list?

Reply:
Line from an older commit. Filters now takes the list of endpoints accordingly